pandas/tests/extension/decimal/array.py

```diff
@@ -153,6 +169,40 @@ def _reduce(self, name, skipna=True, **kwargs):
                                       "the {} operation".format(name))
         return op(axis=0)
 
+    # numpy experimental NEP-18 (opt-in numpy 1.16, enabled in in 1.17)
+    def __array_function__(self, func, types, args, kwargs):
+        def coerce_EA(coll):
```

Comment:
This doesn't seem like a good idea. For now, just use `np.asarray`. I think we have a separate issue about converting EAs to preferred ndarray representation, but `__array__` is the best we have for now.

Reply:
I've already said I'm not thrilled with this approach, but I'm not sure what you're suggesting.
How can you support `pd.concat([EA,EA])` without resorting to something like this?  

`__array__` in relation to what? I've already discussed how implementing NEP-18 on a class, makes numpy ignore `__array__`, since point 4 in he first post.